guides/source/action_mailer_basics.md

```diff
@@ -530,6 +530,8 @@ url helper.
 <%= user_url(@user, host: 'example.com') %>
 ```
 
+NOTE: `non-GET` links don't work when added to mailer templates. They are treated as `GET` links.
```

Comment:
@prathamesh-sonpatki yea that's probably better. You can even link the word to the project https://github.com/rails/jquery-ujs